contrib/completions/zsh/oc

```diff
@@ -5949,6 +5949,8 @@ _oc_adm_router()
     local_nonpersistent_flags+=("--disable-namespace-ownership-check")
     flags+=("--dry-run")
     local_nonpersistent_flags+=("--dry-run")
+    flags+=("--enable-threads")
+    local_nonpersistent_flags+=("--enable-threads")
```

Comment:
You need to re-generate bash completions

Reply:
oh yeah i forgot to run the completions again